### ISSUE TITLE
CNN end to end 

### ISSUE THREAD

**Issue by shantanuo** [Madhu009/Medium_Machine_learning_Blog#4]
I tried the page at Meetups_Notebooks/CNN_End2End/CatsvsDogs.ipynb, but got an error while creating the function "convert_to_tflite". The error is thrown on this line...

`    converter = tf.contrib.lite.TocoConverter.from_frozen_graph(path+'frozen_catsVSdogs45.pb', ['Input'], ['Softmax']])`

May be you have a extra "]" at the end after "softmax".

**Comment by Madhu009** [comment]
yes that was because of extra bracket, remove it and it will will
